spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -50,7 +50,7 @@
       get :metadata
     end
 
-    let(:org_name) { '18F' }
+    let(:org_name) { 'Login.gov' }
```

Comment:
Picking at nits... this should be lower-case "login.gov" to follow branding.

Reply:
updated!